cmd/ak/cmd/sessions/test.go

```diff
@@ -20,8 +21,12 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
+var trimPythonExceptionPath bool
+
+var trimRe = regexp.MustCompile(`\/.*\/ak-(user|runner)-.*?\/`)
+
 var testCmd = common.StandardCommand(&cobra.Command{
-	Use:   "test <txtar-file> [--build-id=...] [--env=...] [--deployment-id=...] [--entrypoint=...] [--quiet] [--timeout DURATION] [--poll-interval DURATION] [--no-timestamps]",
```

Comment:
this shouldn't be an optional flag, this should always be the behavior

Reply:
done